docs/guides/kubernetes-dashboard.md

```diff
@@ -82,7 +82,8 @@ Also, please note that while this guide uses [NGINX Ingress Controller], Pomeriu
 NGINX Ingress controller can be installed via [Helm] from the official charts repository. To install the chart with the release name `helm-nginx-ingress`:
 
 ```bash
-helm repo add stable https://kubernetes-charts.storage.googleapis.com
+# Add kubernetes-dashboard repository
+helm repo add kubernetes-dashboard https://kubernetes.github.io/dashboard/
```

Comment:
All the guides/examples need to be updated to use the new non-stable helm repos, I think...

#1498